branchpro/tests/test_models.py

```diff
@@ -45,6 +45,30 @@ def test_get_serial_intevals(self):
         npt.assert_array_equal(
             br_model.get_serial_intevals(), np.array([1, 2]))
 
+    def test_set_r_profile(self):
+        br_model1 = bp.BranchProModel(0, [1, 2])
+        br_model1.set_r_profile([1], [2])
+        npt.assert_array_equal(br_model1._r_profile, np.array([0, 1]))
```

Comment:
Do we test protected variables? The idea being that they are an implementation detail, while tests are about behaviour?

If we want to expose the current value r_profile what about a method get_r_profile, along the same lines as get_serial_intevals. I'm happy to defer this to a separate issue and PR.